spec/support/transactions.rb

```diff
@@ -65,7 +65,8 @@ class Spec
       # @since 2.6.0
       def initialize(file)
         file = File.new(file)
-        @spec = YAML.load(ERB.new(file.read).result)
+        contents = ERB.new(file.read).result
```

Comment:
Are all of the changes in this file necessary for enabling JRuby testing? It doesn't look like any of them are related

Reply:
Yes they are all necessary.